src/engraving/dom/mscore.h

```diff
@@ -121,6 +121,16 @@ enum class KeySigNatural : char {
     AFTER  = 2                // naturals after accidentals (but always before if going sharps <=> flats)
 };
 
+//---------------------------------------------------------
+//    CourtesyBarlineMode (for key sig. and time sig. changes)
+//---------------------------------------------------------
+
+enum class CourtesyBarlineMode : char {
+    ALWAYS_SINGLE = 0,
+    ALWAYS_DOUBLE = 1,
+    DOUBLE_BEFORE_NEW_SYSTEM = 2,
```

Comment:
How about renaming this value to match the UI (and the more precise meaning)?
```suggestion
    DOUBLE_BEFORE_COURTESY = 2,
```